src/Symfony/Component/Security/Guard/Authenticator/AbstractFormLoginAuthenticator.php

```diff
@@ -41,9 +41,14 @@
      * login page directly), this returns the URL the user should be redirected
      * to after logging in successfully (e.g. your homepage).
      *
+     * @deprecated Implement onAuthenticationFailure() instead of needing this function.
```

Comment:
Should this be "Implement onAuthenticationSuccess() instead of needing this function."?